docs/Using Price Feeds/get-the-latest-price-solana.md

```diff
@@ -2,12 +2,12 @@
 layout: nodes.liquid
 section: smartContract
 date: Last Modified
-title: "Consuming Price Feeds (Solana)"
+title: "Consuming Data Feeds (Solana)"
 permalink: "docs/get-the-latest-price-solana/"
-whatsnext: {"Historical Price Data":"/docs/historical-price-data/", "Price Feeds API Reference":"/docs/price-feeds-api-reference/", "Price Feeds Contract Addresses":"/docs/reference-contracts/"}
+whatsnext: {"Historical Price Data":"/docs/historical-price-data/", "Data Feeds API Reference":"/docs/price-feeds-api-reference/", "Data Feed Contract Addresses":"/docs/reference-contracts/"}
```

Comment:
Should we fix these URLs?

Reply:
We will handle URL changes and redirects in a separate change. There are several.